content/src/editor.less

```diff
@@ -805,19 +818,34 @@ body#pencildoc #diigolet-csm {
 }
 
 .directory .item {
+  text-align: center;
   display: inline-block;
   float: left;
   clear: left;
   text-decoration: none;
-  padding: 0 28px 0 12px;
-  min-height: 28px;
   color: #222;
-  cursor: pointer
+  cursor: pointer;
+  div {
+    width: 200px;
```

Comment:
(The reasoning for 154px is that it allows 4 columns on the common screen width 1280px.)

We might need to shorten text like "Create new file..." to just "New file...".